src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int bound = 100;
+        Random numberBall = new Random();
```

Comment:
```suggestion
        Random random = new Random();
```

те саме, змінну random треба винести на рівень класу, що б не створювати кожен раз екземпляр цього класу при вилику методу getRandomColor()